### PR TITLE
Updated about and career page to stress importance of diversity to Gitpod ✊ 🌈

### PR DESCRIPTION
**Pull request by JohannesLandgraf**

### CODE REVIEW

**Comment by gtsiolis** [comment]
Taking a look now! 👀

File: src/pages/careers.tsx [hunk 1]
```diff
@@ -85,26 +85,26 @@ const StyledCareersPage = styled.div`
...
+        title: "Open",
+        text: "We are open minded, transparent and curious. We remain students of the game, not masters of the game.",
         img: Line,
         alt: 'Line'
     },
     {
-        title: "People Who Care",
-        text: "We support each other and grow to new strengths together. On regular team events, we’re empowering this bond.",
+        title: "Developer-first",
+        text: "Developers architect our future. We are developers ourselves and built Gitpod to make lifes of  developers easier.",
```
Comment (gtsiolis) [inline src/pages/careers.tsx:95]: **nitpick**: Small typo.

```suggestion
        text: "Developers architect our future. We are developers ourselves and built Gitpod to make lives of developers easier.",
```

File: src/pages/careers.tsx [hunk 2]
```diff
@@ -217,9 +217,9 @@ const Careers = () => (
                             <img src={Team} alt="Gitpod Team" />
                         </div>
                         <div className="text">
-                            <p>Come join a young tech-driven team, with the mission of making software development better for everyone, and strong values around openness, honesty, and effective pragmatism. </p>
-                            <p>We're convinced that continuous learning and exploring is the way to build the best solutions sustainably.</p>
-                            <p><strong>Help us shape Gitpod’s future from Berlin, Paris, or anywhere in the world!</strong></p>
+                            <p>Come join a quickly growing venture-backed team and work together with incredible humans from 🇦🇺 🇧🇷 🇨🇦 🇫🇷 🇩🇪 🇬🇷 🇵🇰 🇷🇺 🇺🇸  - in an environment with strong values around openness, honesty, and diversity. </p>
+                            <p> Unfortunately, we are not as diverse as we would like to be.</p>
```
Comment (gtsiolis) [inline src/pages/careers.tsx:221]: **thought**: If there was one thing I would change in this PR is to skip the two references of the word _unfortunately_. I think the text can suffice without it and doesn't add much value. What do you think? 😃

File: src/pages/about.tsx [hunk 3]
```diff
@@ -234,23 +236,21 @@ const AboutPage: React.SFC<{}> = () => (
...
                             </div>
                             <div className="about__box">
-                                <img alt="TypeFox Team" src={Team} />
-                                <h3>About the Team</h3>
-                                <p>We're a small tech-driven team with collectively over 50 years of experience in language design, IDEs, and tool development and with strong values around openness, honesty, and effective pragmatism. If you're interested in joining us, please have a look at our <Link to="/careers">Job Openings</Link>.</p>
+                                <img alt="Wallpaper1" src={gitpodWallpaper1} />
```
Comment (gtsiolis) [inline src/pages/about.tsx:244]: **praise**: That's a good photo. 📸

File: src/pages/about.tsx [hunk 4]
```diff
@@ -234,23 +236,21 @@ const AboutPage: React.SFC<{}> = () => (
...
-                                <p>We are a driving force behind the language server protocol and the initiators of Eclipse Theia, the next generation IDE that powers Gitpod.</p>
+                                <img alt="Team" src={Team} />
+                                <h3>⚓️ About Gitpod</h3>
+                                <p>We have built developer tools, created programming languages and successfully grew open source communities over the last 10 years. We are developers ourselves - with Gitpod we want to capitalize on our learnings and build a company that simplifies the lifes of all developers. Experienced international investors from the world of developer tools and open source <Link to="https://www.crunchbase.com/organization/gitpod">support us</Link> on our journey. <br /> <br /> We are a fully distributed team with people from all over the world 🇦🇺 🇧🇷 🇨🇦 🇫🇷 🇩🇪 🇬🇷 🇵🇰 🇷🇺 🇺🇸 </p>
                             </div>
                             <div className="about__box">
-                                <img alt="TypeFox Team" src={Team} />
```
Comment (gtsiolis) [inline src/pages/about.tsx:243]: Go Kiel! 🏀

**Review by gtsiolis — COMMENTED** [review]
Now that's a great MVC (Minimum Viable Change)! Thanks for making these changes @JohannesLandgraf! Looks good, left some minor comments.

File: src/pages/about.tsx (hunk 3 above)
Comment (JohannesLandgraf) [reply src/pages/about.tsx:244]: Yeah some dude from Athens took that ¯\_(ツ)_/¯

File: src/pages/about.tsx (hunk 4 above)
Comment (JohannesLandgraf) [reply src/pages/about.tsx:243]: So much happened since that picture 🤯✨

File: src/pages/careers.tsx (hunk 2 above)
Comment (JohannesLandgraf) [reply src/pages/careers.tsx]: Agree, will change!

File: src/pages/careers.tsx [hunk 5]
```diff
@@ -217,9 +217,9 @@ const Careers = () => (
                             <img src={Team} alt="Gitpod Team" />
                         </div>
                         <div className="text">
-                            <p>Come join a young tech-driven team, with the mission of making software development better for everyone, and strong values around openness, honesty, and effective pragmatism. </p>
-                            <p>We're convinced that continuous learning and exploring is the way to build the best solutions sustainably.</p>
-                            <p><strong>Help us shape Gitpod’s future from Berlin, Paris, or anywhere in the world!</strong></p>
+                            <p>Come join a quickly growing venture-backed team and work together with incredible humans from 🇦🇺 🇧🇷 🇨🇦 🇫🇷 🇩🇪 🇬🇷 🇵🇰 🇷🇺 🇺🇸  - in an environment with strong values around openness, honesty, and diversity. </p>
```
[the remark this answers is not in the document]
Comment (JohannesLandgraf) [reply src/pages/careers.tsx:220]: Not yet public (sry community) 🙊 Will tell you via DM

**Pull request merged**